libs/net/dal/Migrations/1.0.37/Up/PostUp/00-Report.sql

```diff
@@ -0,0 +1,125 @@
+DO $$
+BEGIN
+
+UPDATE public.report SET
```

Comment:
This won't work.  Look at the latest database schema changes.  templates have been extracted to `ReportTemplate`

Reply:
Updated, and thanks for the good catch.